docs(llm): proofread (#2905)
scaleway/docs-content: ai-data/llm-inference/reference-content/llama-2-70b.mdx

## How to use it

### Sending Inference requests

> To perform inference tasks with your LLM model deployed at Scaleway, follow these steps:

Reviewer: ```suggestion
To perform inference tasks with your LLM model deployed at Scaleway, use the following command:
```

### Receiving Inference responses

Upon sending the HTTP request to the public or private endpoints exposed by the server, you will receive inference responses from the managed Inference server. 
Process the output data according to your application's needs. The response will contain the output generated by the LLM model based on the input provided in the request.

<Message type="note">

>   Despite efforts for accuracy, the possibility of generated text containing inaccuracies or 'hallucinations' exists. Always independently verify the content generated.

Reviewer 2: ```suggestion
  Despite efforts to ensure accuracy, the possibility of generated text containing inaccuracies or 'hallucinations' exists. Always independently verify the content generated.
```
